HumanControls/DisplayController.cpp

```diff
@@ -7,8 +7,14 @@ DisplayController::DisplayController(unsigned int displayAddress, unsigned int d
 
 void DisplayController::init()
 {
+    uint8_t down[8] = {0x4, 0x4, 0x4, 0x4, 0x4, 0x1f, 0xe, 0x4};
```

Comment:
Make this a constant

Reply:
It can be, but it is not necessary. The library stores the custom char.